tasks/eth/011-proto-ver+ownership-transfers/.env

```diff
@@ -0,0 +1,4 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+FOUNDATION_OP_SAFE=0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A
+OWNER_SAFE=0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A
+FOUNDATION_SAFE=0x847B5c174615B1B7fDF770882256e2D3E95b9D92
```

Comment:
Can we give better names, or use comments, to disambiguate `FOUNDATION_OP_SAFE` and `FOUNDATION_SAFE`?

Reply:
added `_UP`